cli/cmd/cluster-apply.go

```diff
@@ -34,8 +34,10 @@ var (
 
 var clusterApplyCmd = &cobra.Command{
 	Use:   "apply",
-	Short: "Apply configuration changes to a Lokomotive cluster with components",
-	Run:   runClusterApply,
+	Short: "Deploy or update a Lokomotive cluster",
+	Long: `Deploy or update a Lokomotive cluster.
+Unless explicitly skipped, Lokomotive components listed in the configuration are applied as well.`,
+	Run: runClusterApply,
```

Comment:
nitpick: The commit title only mentions the short description was changed.

Reply:
Thanks, fixed.